src/R/Core/Test/Tokens/TokenizeIdentifierTest.cs

```diff
@@ -63,15 +64,15 @@ public class TokenizeIdentifierTest : TokenizeTestBase<RToken, RTokenType> {
             }, (token, tokenType) => token.TokenType == tokenType);
         }
 
-        [Test]
-        public void TokenizeIdentifierLogicalTest02() {
-            var tokens = Tokenize("1 <- F", new RTokenizer());
-
-            tokens.Should().Equal(new [] {
-                RTokenType.Number,
-                RTokenType.Operator,
-                RTokenType.Logical,
-            }, (token, tokenType) => token.TokenType == tokenType);
+        [CompositeTest]
+        [InlineData("1 <- F", 2, RTokenType.Logical)]
+        [InlineData("F <- 1", 0, RTokenType.Identifier)]
```

Comment:
This is specifically to tread T/F as identifier in certain expressions, but sure.